### DISCUSSION TITLE
Edit table data directly on Backend Azure SQL

### DISCUSSION THREAD

**Discussion by michaelcoelsch** [Azure/azure-mobile-apps#756] [Q&A]
Hi,
i implemented Azure Mobile Apps in my WinUI App and it's working perfectly.

But when I edit records in my SQL Backend table directly, sync isn't working and the client SQLite is not updated. I think, the reason is, that when I update a record directly in an Azure SQL table, the fields [CreatedAt] and [Version] stay unchanged.
Is there a possibility to use Triggers in my backend SQL tables to update above mentioned fields after editing a field?

Thanks

**Comment by adrianhall** [comment]
Yes - absolutely.  You only really need to change the Version - the CreatedAt field is no longer used during sync.

**Reply by adrianhall** [reply]
Also, change the UpdatedAt field!
